silk/tests/hwconfig.ini

```diff
@@ -1,12 +1,17 @@
 # Sample HW config file
 
-#Nordic dev board as a test device
+# Global settings
+[DEFAULT]
+ClusterID: 0
+
+# Nordic dev board as a test device
 [Dev-6E83]
 HwModel: Nrf52840
 HwRev: 1.0
 InterfaceSerialNumber: C83FBBC96E83
 USBInterfaceNumber: 1
 DutSerial: 683830894
+OTNSVisPosition: 200, 200
 
 # Nordic dev board as a Thread Sniffer
 [Dev-6489]
```

Comment:
missing OTNSVisPosition for Dev-6489

Reply:
Does sniffer needs a position? I thought sniffer doesn't show up as part of the test case.